spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -45,16 +43,10 @@
 from spinn_front_end_common.interface.provenance import (
     ProvidesProvenanceDataFromMachineImpl)
 from spynnaker.pyNN.data import SpynnakerDataView
-from spynnaker.pyNN.models.common import (
-    AbstractSpikeRecordable, AbstractNeuronRecordable, AbstractEventRecordable,
-    NeuronRecorder)
+from spynnaker.pyNN.models.common import NeuronRecorder
 from spynnaker.pyNN.models.abstract_models import (
-    AbstractPopulationInitializable, AbstractAcceptsIncomingSynapses,
-    AbstractPopulationSettable, AbstractContainsUnits, AbstractMaxSpikes,
-    HasSynapses, SupportsStructure)
-from spynnaker.pyNN.exceptions import InvalidParameterType, SpynnakerException
-from spynnaker.pyNN.utilities.ranged import (
```

Comment:
Last known usage of SpynnakerRangeDictionary removed but class still exists

Reply:
Removed